app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -241,8 +262,8 @@ public static int nthToLastCharIndex(int elementNumber, String str, char element
      * Formats input to plain mm:ss format
      */
     public static String formatTimer(long timer) {
-        final long min = TimeUnit.SECONDS.toMinutes(timer);
-        final long sec = TimeUnit.SECONDS.toSeconds(timer - TimeUnit.MINUTES.toMillis(min));
```

Comment:
It seems that the timer was already in seconds. The mistake seems to be the `toMillis(min)`, it should be `toSeconds(min)`